crates/xilem_html/web_examples/todomvc/index.html

```diff
@@ -0,0 +1,537 @@
+<!DOCTYPE html>
```

Comment:
This looks like it came from somewhere. Does it need attribution?

Reply:
I've added a note to the example's README.